### PR TITLE
add html css img

### PR DESCRIPTION
**Pull request by GitSP-arch**
create first version of desktop template

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,177 @@
+
+
+  body
+  {
+    margin:0 0 0 4%;
+    margin-right:4%;
+  }
```
Comment (remi-boivin) [inline css/index.css:7]: It a shit

File: index.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+    <body>
+        <header class="section-header">
+            <img src="./img/logo/Booki.png" id="Booki" alt="Booki"/>
+            <div class="colonne-menu">
+                <div class="bordure-hebergement"></div> 
+                <a href="#Hébergements à Marseille"> Hébergements</a>
+                <a href="#Activités à Marseille"> Activités</a>
+                <div class="marge-menu"></div>
+            </div>
```
Comment (remi-boivin) [inline index.html:20]: You did well @GitSP-arch

**Review by remi-boivin — CHANGES_REQUESTED** [review]
You must chance section-header

File: index.html (hunk 2 above)
Comment (GitSP-arch) [reply index.html:20]: ok

File: css/index.css (hunk 1 above)
Comment (GitSP-arch) [reply css/index.css:7]: well

**Pull request merged**